Assets/Scripts/Timer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Timer : MonoBehaviour {
+    
+    public float TimeLeft;

```

Comment:
Would it be better to make `TimeLeft` an integer and cast when subtracting the delta instead? Would it help readability?